H5P.BranchedVideo/branched-video.js

```diff
@@ -47,6 +47,69 @@ H5P.BranchedVideo = (function ($) {
       this.currentTime = 0.00;
       this.watchedTime = 0.00;
       this.nodes = [];  // array of node objects
+
+      this.playedTime = [[0,0]];
+
+      this.getPlayedTime = function(){
+        return this.playedTime;
+      }
+
+      this.updatePlayedTime = function(time){
```

Comment:
Could you please add some comments to all your functions to explain what they do?